### PR TITLE
Jobs : Post, edit, delete

### PR DESCRIPTION
**Pull request by Melody-Le**
#11 need to be merged before this PR merge

### CODE REVIEW

File: controllers/job_controller.js [hunk 1]
```diff
@@ -1,82 +1,80 @@
+const jobs = require("../models/jobs.js");
```
Comment (tlkiong) [inline controllers/job_controller.js]: ???

File: controllers/job_controller.js [hunk 2]
```diff
@@ -1,82 +1,80 @@
...
     } catch (error) {
       console.log(error.message);
     }
   },
-  // //Method GET
-  // newProjectForm(req, res) {
-  //   res.render("./projects/project_new.ejs");
-  // },
+  //Method GET
+  new(req, res) {
```
Comment (tlkiong) [inline controllers/job_controller.js]: Why suddenly your naming convention change?

you didn’t name the top function as show

File: views/profiles/index.ejs [hunk 3]
```diff
@@ -27,25 +27,29 @@
...
-            <h6>Skill</h6>
-            <ul class=" container list-inline px-0 mb-0">
-              <% profileOwner?.skills.forEach((skill) => { %>
-                <li class="list-inline-item mx-0 rounded"><%= skill%></li>
-              <% }) %>
-            </ul>
-          </div>
+          <% if (profileOwner.isFreelancer) { %>
+            <div>
+              <a href="<%= profileOwner.linkedIn%>">LindedIn Link</a>
```
Comment (tlkiong) [inline views/profiles/index.ejs]: Spacings. Check LinkedIn variable.

Comment (Melody-Le) [reply views/profiles/index.ejs]: yeah, i had mis-typing here. But what you when say: check LinkedIn Variable? 
I have just check, It work

File: controllers/job_controller.js (hunk 1 above)
Comment (Melody-Le) [reply controllers/job_controller.js]: my mistake only. I removed it.

File: controllers/job_controller.js [hunk 4]
```diff
@@ -5,78 +5,75 @@ const controller = {
...
-  // //Method POST: to create new project
-  // async create(req, res) {
-  //   const projectData = req.body;
-  //   const profileOwner = await Users.findById(`${req.session?.currentUser?._id}`);
-  //   projectData.author_id = profileOwner._id;
-  //   const newProject = await Projects.create(projectData);
-  //   res.redirect(`/projects/${newProject._id}`);
-  // },
+  //Method POST: to create new job
+  async create(req, res) {
```
Comment (tlkiong) [inline controllers/job_controller.js]: please check everywhere. don't just check at one point. this method and others too.

remember rule #1: consistency

File: views/jobs/index.ejs [hunk 5]
```diff
@@ -34,9 +34,11 @@
           <div class="mt-3">
             <h6>Provided Skill</h6>
             <ul class="container list-inline px-0">
-              <% job.skills.forEach((skill) => { %>
-                <li class="list-inline-item mx-0 rounded"><%= skill %></li>
-              <% }) %>
+              <% if (job.skills.length) { %>
```
Comment (tlkiong) [inline views/jobs/index.ejs]: Do you even need this line?

File: views/profiles/index.ejs (hunk 3 above)
Comment (tlkiong) [reply views/profiles/index.ejs]: i meant the mistype

**Review by tlkiong — APPROVED** [review]

**Pull request merged**